x-pack/test/security_api_integration/tests/session_concurrent_limit/cleanup.ts

```diff
@@ -371,17 +371,17 @@ export default function ({ getService }: FtrProviderContext) {
         unauthenticatedSessionOne.cookie,
         unauthenticatedSessionOne.location
       );
-      await setTimeoutAsync(500);
       const samlSessionCookieTwo = await finishSAMLHandshake(
         unauthenticatedSessionTwo.cookie,
         unauthenticatedSessionTwo.location
       );
-      await setTimeoutAsync(500);
```

Comment:
__nit:__ I think it'd still be safer to keep these `setTimeoutAsync` to make sure that `samlSessionCookieOne` has the oldest timestamp. In certain circumstances (under unusual load on CI for example) it might be possible for the second request to get a bit older timestamp.